drivers/net/macsec.c

```diff
@@ -933,7 +933,15 @@ static enum rx_handler_result handle_not_macsec(struct sk_buff *skb)
 		 * SecTAG, so we have to deduce which port to deliver to.
 		 */
 		if (macsec_get_ops(macsec, NULL) && netif_running(ndev)) {
-			if (ether_addr_equal_64bits(hdr->h_dest, ndev->dev_addr)) {
+			if (ndev->flags & IFF_PROMISC) {
+				nskb = skb_clone(skb, GFP_ATOMIC);
```

Comment:
Why not divert the packet? We should deliver is to macsec device only, if I'm not wrong...

Reply:
that is promiscuous mode - all receive everything

Reply:
Ah, okay.